src/main/java/org/entando/kubernetes/model/bundle/descriptor/plugin/PluginDescriptorV1Spec.java

```diff
@@ -0,0 +1,21 @@
+package org.entando.kubernetes.model.bundle.descriptor.plugin;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
```

Comment:
If the class is meant to be immutable consider using `@Value` instead of `@Data`

Reply:
It could be the rightest way but we are using jackson to parse the descriptors. Jackson should use reflection and therefore the empty constructor and then the setters. I think it's impossible to apply the immutable pattern using these libs